tracing-attributes/src/lib.rs

```diff
@@ -795,7 +798,7 @@ mod kw {
 // able to use that information to instrument the proper function.
 // (this follows the approach suggested in
 // https://github.com/dtolnay/async-trait/issues/45#issuecomment-571245673)
-fn get_async_trait_name(block: &Block, block_is_async: bool) -> Option<String> {
+fn get_async_trait_function(block: &Block, block_is_async: bool) -> Option<&ItemFn> {
```

Comment:
Nit: I think that, now that this function returns a function item rather than _just_ the function's name, the comment on it ought to be updated...